scripts/main.js

```diff
@@ -71,4 +71,94 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+
+// Opdracht 1 - Gegevens aanspreken
+
+console.log('De afdeling Sales heeft ' + departments.sales.numberOfEmployees + ' medewerkers');
+console.log('Marketing is een leuke afdeling om te werken. ' +departments.marketing.description);
+console.log('De afdeling Customer Service heeft ' +departments["customer-service"].numberOfEmployees + ' medewerkers');
+console.log('Sales is een uitdagende afdeling om te werken als Verkoopmanager.  ' +departments.sales.description);
+
+// Opdracht 2 - prompten en beslissen
+
+let gekozenAfdeling = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]');
```

Comment:
in de _uitwerkingen_ wordt er geen let-variabele gebruikt, maar **const**. Ik denk dat dit is omdat je per gebruik deze variabele niet meer wijzigd.